<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyProfile</class>
 <widget class="QWidget" name="MyProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Muprofile</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:
rgb(204, 235, 255)</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>1151</width>
     <height>761</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="follower">
   <property name="geometry">
    <rect>
     <x>1400</x>
     <y>10</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 19, 124, 255), stop:0.295775 rgba(29, 44, 177, 255), stop:0.591549 rgba(80, 89, 207, 255), stop:1 rgba(79, 165, 189, 255))</string>
   </property>
   <property name="text">
    <string>see followers</string>
   </property>
  </widget>
  <widget class="QPushButton" name="following">
   <property name="geometry">
    <rect>
     <x>1420</x>
     <y>430</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 19, 124, 255), stop:0.295775 rgba(29, 44, 177, 255), stop:0.591549 rgba(80, 89, 207, 255), stop:1 rgba(79, 165, 189, 255))</string>
   </property>
   <property name="text">
    <string>see following</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_2">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>50</y>
     <width>541</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_3">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>450</y>
     <width>541</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>120</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>120</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>70</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Followings</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>70</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Followers</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>840</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 19, 124, 255), stop:0.295775 rgba(29, 44, 177, 255), stop:0.591549 rgba(80, 89, 207, 255), stop:1 rgba(79, 165, 189, 255))</string>
   </property>
   <property name="text">
    <string>&lt;&lt;Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>1941</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="profile">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>20</y>
     <width>211</width>
     <height>251</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>listWidget_4</zorder>
  <zorder>label_5</zorder>
  <zorder>listWidget</zorder>
  <zorder>listWidget_2</zorder>
  <zorder>listWidget_3</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>follower</zorder>
  <zorder>following</zorder>
  <zorder>pushButton</zorder>
  <zorder>profile</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
